src/ngraph/pass/constant_folding_broadcast.cpp

```diff
@@ -45,11 +45,11 @@ shared_ptr<op::Constant> fold_constant_broadcast(shared_ptr<op::Constant> consta
         auto static_bcast_axes = broadcast_v1->get_broadcast_axes();
         if (static_bcast_axes.first)
         {
-            runtime::reference::broadcast<T>(constant->get_data_ptr<T>(),
-                                             data_ptr,
-                                             constant->get_shape(),
-                                             out_shape,
-                                             static_bcast_axes.second);
+            runtime::gcpu::kernel::broadcast<T>(constant->get_data_ptr<T>(),
```

Comment:
These should be moved out of gcpu and into core if they are to be used in core.

Reply:
Yes, this is a good idea, moved to opt_kernel